packages/yoga/src/Card/native/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import styled from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+import { View } from 'react-native';
+import { Clock } from '@gympass/yoga-icons';
+
+import Card from '../Card';
+import Text from '../../../Text';
+
+const Event = styled(Card)`
+  flex-direction: row;
+
+  width: 280px;
+  height: 104px;
```

Comment:
Is this right?

Reply:
Yes, but just the `width`, the height I've adjusted to *auto*